app/models/template.rb

```diff
@@ -1,5 +1,6 @@
 class Template < ActiveRecord::Base
   include Exportable
+  attr_accessor :modify_locked, :modify_default
```

Comment:
yeah I could do that but I though people might find it useful e.g. in tests when they just disable locking in setup and re-enable in teardown even though the `ignore_locking` is preferable elsewhere